plugins/kubernetes-react/src/translation.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createTranslationRef } from '@backstage/core-plugin-api/alpha';
+
+/** @alpha */
+export const kubernetesReactTranslationRef = createTranslationRef({
```

Comment:
Ah, you'll want to export this via the `/alpha` entry point too, otherwise you don't be able to configure the message using the ref.

Reply:
🤔 how exactly? do you have examples?

Reply:
I guess I'm missing an important key point here on how to customize the button text without using the new key I added in the `app-config`

Reply:
Ideally https://backstage.io/docs/plugins/internationalization#for-an-application-developer-overwrite-plugin-messages explains it all, but just in case, you'd do something like this:

```ts
import { kubernetesReactTranslationRef } from '@backstage/plugin-kubernetes-react/alpha';

const app = createApp({
  // ...
  __experimentalTranslations: {
    resources: [
      createTranslationMessages({
        ref: kubernetesReactTranslationRef,
        messages: {
          "podDrawer.buttons.delete": 'Restart Pod'
        }
      })
    ]
  }
  // ...
});
```

Reply:
For the `/alpha` export, see https://backstage.io/docs/tooling/cli/build-system#subpath-exports